src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, Drawable {

```

Comment:
this class should be abstract
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {

```